const_cast.md

```diff
@@ -0,0 +1,33 @@
+## const_cast conversion
+const_cast is an explicit cast operator that is mainly used to remove (or add) constness from/to a pointer or reference.
+This operator is only able to remove or add the constness to **what a pointer (or reference) points to**, and *never* the constness of the object itself: otherwise, we would break the immutability of the object.
+
+## Examples
+- Remove constness:
+```cpp
+int main() {
```

Comment:
Reduce LOC by removing `int main()`, we don't need a whole program to demonstrate how it works